lib/cdo/lesson_import_helper.rb

```diff
@@ -102,51 +96,45 @@ def self.create_activity_sections(activity_markdown, lesson_activity_id, levels)
 
     sections = []
     name = ''
-    position = 1
```

Comment:
I noticed position was removed. Don't need it anymore?

Reply:
They're regenerated down on line 150 so I decided this wasn't serving a purpose anymore. Activity sections need a position in order to be saved so I changed it to always set the position to 0 on line 128. I'm happy to switch back here though!